src/ai/backend/manager/services/users/actions/delete_user.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from typing import Optional, override
+
+from ai.backend.manager.actions.action import BaseActionResult
+from ai.backend.manager.services.users.actions.base import UserAction
+
+
+@dataclass
+class DeleteUserAction(UserAction):
+    email: str
+
+    @override
+    def entity_id(self) -> Optional[str]:
+        return self.email
+
+    @override
+    def operation_type(self) -> str:
+        return "delete"
```

Comment:
I think we should explicitly include entity name in `operation_type` "user", like in `delete_user`.

Right now, there's only one `xxx_user` action, so even if we just write `xxx`, it's clear that it refers to a `xxx_user` action. However, I think batch actions might be added later. 

Thoughts? @HyeockJinKim

Reply:
As a result of the discussion, we have established the following convention

Do not include the entity name in `operation_type`, but for BatchAction, add the suffix "_multi".